src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,28 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo + System.lineSeparator());
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate startDay = LocalDate.parse(dateFrom, dateTimeFormatter).minusDays(1);
+        LocalDate endDay = LocalDate.parse(dateTo, dateTimeFormatter).plusDays(1);
+        int salary;
+        for (String name : names) {
+            salary = 0;
```

Comment:
Don't need to introduce it before loop, since you are using this variable only in loop block